homeassistant/components/sensor/zwave.py

```diff
@@ -56,8 +60,17 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         # Check workaround mappings for specific devices.
         if specific_sensor_key in DEVICE_MAPPINGS:
             if DEVICE_MAPPINGS[specific_sensor_key] == WORKAROUND_IGNORE:
+                logger.info('Ignoring device: %s, Index: %s',
+                            zwave._value_name(value), value.index)
                 return
 
+    if value.label == 'Previous Reading':
+        # Power measurement devices may export a 'Previous Reading' value
+        # This can be problematic, as can be seen here:
+        # https://github.com/home-assistant/home-assistant/issues/3509
+        logger.info('Ignoring device: %s', zwave._value_name(value))
```

Comment:
@devdelay you mean that we preserve the "previous reading" values? In my opinion, we don't need those in home-assistant. We have better way to deal with old states - there is the new 'stats' component, which is even independent of the used platform. And the automation templates can also access the old state.